lib/source/metadata/parser.js

```diff
@@ -116,6 +116,7 @@ Parser.mappings = {
       return;
     }
 
+    properties['iam-role'] = roles[0].substr(path.length);
```

Comment:
Opt'd for `Path.basename` in d27ba65. We use Path's join in other places when interacting with meta-data keys.